src/pages/api/user/match-history.ts

```diff
@@ -67,61 +68,26 @@ export default async function handler(
 
   if (resultPlayerOne.Items) {
     matchDataPlayerOne = resultPlayerOne.Items.map((item: any) => ({
-      id: item.MATCH_ID.N,
-      player: item.TEAM_1.S,
-      opponent: item.TEAM_2.S,
-      outcome: item.OUTCOME.S,
-      type: item.MATCH_TYPE.S,
-      replay: item.REPLAY_URL ? item.REPLAY_URL.S : null,
-      status: item.MATCH_STATUS.S,
+      id: item.match_id.N,
+      player: teamname,
+      opponent: item.players.L[0].M.current.BOOL ? item.players.L[1].M.teamName.S : item.players.L[0].M.teamName.S,
+      outcome: item.placement ? item.placement.N.toString(): "PENDING",
+      type: item.category.S,
+      replay: item.s3_key ? process.env.S3_URL_TEMPLATE + item.s3_key.S : null,
```

Comment:
will get another s3_url_template for matches. so need to add another env variable for that